src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('Roboto-Medium.ttf');
+}
+
+html {
+  --brand-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100dvw;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.header__list {
+  list-style: none;
+  display: flex;
+}
+
+.header__item {
+  position: relative;
+  height: 60px;
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.header__item:last-child {
+  margin: 0;
+}
+
+.header__link {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  text-decoration: none;
+  color: black;
+  height: 100%;
+}
+
+.header__link:hover {
+  color: var(--brand-color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: ' ';
```

Comment:
```suggestion
  content: '';
```

Reply:
Thought pseudo elements should always have content to be displayed, but it was about content property not the content value. Changed.